packages/react-reconciler/src/ReactFiberWorkLoop.old.js

```diff
@@ -1467,6 +1467,9 @@ export function popRenderLanes(fiber: Fiber) {
 function prepareFreshStack(root: FiberRoot, lanes: Lanes): Fiber {
   root.finishedWork = null;
   root.finishedLanes = NoLanes;
+  if (root.incompleteTransitions === null) {
```

Comment:
Might as well make this a non-nullable type, rather than create a new one on every render